tests/integration/common/auth/test_auth.py

```diff
@@ -378,6 +378,31 @@ def test_platform_admin_remove_all_other_roles(self, anonymous_client, factories
 
         assert response.status_code == 200
 
+    def test_platform_admin_signin_claims_pending_invitations(self, anonymous_client, factories, db_session):
+        with patch("app.common.auth.build_msal_app") as mock_build_msal_app:
```

Comment:
nit: bit of an overzealous mocking - I think it's nice to only mock the bits of code that expect it so that, as a reader, I know what it's actually being used for.

Lines 383-386 don't use it, 388-397 do, 398-402 don't?